src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -67,6 +68,8 @@ function AnchorRenderer({tnode, key, style}) {
 
     // An auth token is needed to download Expensify chat attachments
     const isAttachment = Boolean(htmlAttribs['data-expensify-source']);
+    const fileName = isAttachment ? lodashGet(tnode, 'domNode.children[0].data') : null;
```

Comment:
Add a default param to  `lodashGet(tnode, 'domNode.children[0].data')` of an empty string '' and remove the conditional and just use that

Reply:
Done